build-web/build.sh

```diff
@@ -10,14 +10,14 @@ export NUM_JOBS=5
 declare -a JOBS=(
   "target=template_debug"
   "target=template_release"
-  "target=template_debug dlink_enabled=yes"
-  "target=template_release dlink_enabled=yes"
+  "target=template_debug dlink_enabled=yes module_dotnet_enabled=yes"
+  "target=template_release dlink_enabled=yes module_dotnet_enabled=yes"
```

Comment:
I don't expect Web to work just yet, but if/when it does, I assume it would only work for the `dlink_enabled` builds (i.e. GDExtension support).

Reply:
Actually, the WIP PR I made requires the opposite. It didn't work with `dlink_enabled` because too many symbols or something like that. But we don't need dynamic linking because Mono is linked statically.

We may need the statically-linked Mono template to be separate if we don't want that to be included in GDScript-only projects.